extensions/wikia/AdEngine/js/AdLogicPageParams.js

```diff
@@ -210,7 +203,6 @@ define('ext.wikia.adEngine.adLogicPageParams', [
 		}
 
 		extend(params, decodeLegacyDartParams(targeting.wikiCustomKeyValues));
-		overrideEsrbRating(params);
```

Comment:
or `decodeLegacyDartParams` should skip `esrb` occurrences (to make it safer in future)